tdvt/tdvt/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.4.0'
+__version__ = '2.5.0b1'
```

Comment:
Should this just be 2.5.0?

Reply:
since this was not major change Luke suggested we name it differently

Reply:
Since we don't have perms to overwrite packages in Artifactory, I've been packaging and uploading w/a beta tag so they can be tested in a TC run. Once you get a clean TC run it should be packaged/uploaded without the `b`. 😄